cmd/cluster-olm-operator/main.go

```diff
@@ -75,6 +75,7 @@ func newStartCommand() *cobra.Command {
 		"cluster-olm-operator",
 		version.Get(),
 		runOperator,
+		nil,
```

Comment:
Does this look right? 

The documentation for clock.Clock [here](https://github.com/openshift/library-go/blob/master/pkg/controller/controllercmd/cmd.go#L81) reads 

```
// Clock allows for injecting fake or real clocks into code that
// needs to do arbitrary things based on time.
```

Reply:
`nil` is probably find here and below.

Reply:
I did 
```golang
clock.RealClock{}
```
but I was literally cargo-culting another example which did nothing with the resulting clock.

Reply:
It is useful to look through the change that introduced this new function parameter to understand what is expected. https://github.com/openshift/library-go/pull/1880

TL;DR (but I really would encourage reviewing the details in this case and others when things like this arise): we need to pass a non-nil clock. `clock.RealClock{}` seems like the correct choice in `main.go`

Reply:
Yeah, that's [where](https://github.com/openshift/library-go/pull/1880/files#diff-8a3d8d95d4d7548a8fcf8e71fd086374dd3098542f86d7440f41ed42eb974222R38) I cargo-culted it from since I figured it would do no harm but wasn't 100% on why it was desirable. 😅

Reply:
Mea culpa, between the go version sneakily updating behind the scenes and all the slack messages, I was totally trying to rush this in unnecessarily 😅 let me see if I can find some explanation to confirm `clock.RealClock{}` is the right answer here.

Reply:
Update: `clock.RealClock{}` is needed otherwise a nil pointer error is thrown.